ruby/rollbar.rb

```diff
@@ -0,0 +1,10 @@
+Rollbar.configure do |config|
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```

Comment:
Ik vraag me af of dat zo is. Volgens mij moet een app prima kunnen draaien zonder Rollbar en is het een keuze om dat te gebruiken. Het is een beetje raar om wel al de config erin te zetten en het nog niet actief te gebruiken, maar het zou in theorie moeten kunnen wat mij betreft.